index.html

```diff
@@ -16,11 +16,22 @@
       href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/atom-one-dark.min.css"
     />
   </head>
+
   <body>
+  
     <div class="container">
-   <section id="alert">
+     <section>
+       <h2 class="h2">Rating </h2>
+      <pre>
+        <code class="language-html">
+          <div class="stars" style="--rating:3.5;"></div>
+        </code>
+      </pre> 
+      <div class="stars" style="--rating:3.5; background-color: rebeccapurple;"></div>
```

Comment:
Could you please explain why you used the inline styling when you have also used class?